docs/releasing.md

```diff
@@ -83,6 +88,11 @@ This triggers two things:
   [Quay tags page](https://quay.io/repository/metal3-io/ip-address-manager?tab=tags).
   If the release tag build is not visible, check if the action has failed and
   retrigger as necessary.
+- GH job `create draft release` creates draft release. Don't publish the release
+  until release tag is visible in
+  [Quay tags page](https://quay.io/repository/metal3-io/ip-address-manager?tab=tags).
+  If the release tag build is not visible, check if the action has failed and
+  retrigger as necessary.
 
 We also need to create one or more tags for the Go modules ecosystem:
```

Comment:
This is also obsolete now, or needs to be reworded in a way that explains how the new automation does it.